src/application-error-boundary/ApplicationErrorBoundary.jsx

```diff
@@ -67,6 +67,10 @@ class ApplicationErrorBoundary extends React.Component {
 
   componentDidUpdate() {
     this.resetError();
+
+    if (this.errorBoundaryRef.current) {
+      this.errorBoundaryRef.current.focus();
```

Comment:
Moving focus on every update is likely too often. The error boundary may be visible while the user interacts with other parts of the app (like the navigation header), and we don't want to move focus when those updates trickle down to the error boundary.

Can we limit this focus logic to just the initial presentation? I think _ideally_ we'd only move focus if the item that was previously focused is removed as part of the status view presentation, but I don't know if we can determine that reliably.

Reply:
Thinking about this some more, I'm not sure if we need to be moving focus at all. Using the "alert" role might be enough to ensure the error is read and accessible. Setting focus to the error may make sense if the previously focused element was removed during the error presentation, but if not, the focus moving may be more obtrusive than we expect.

Reply:
Removed focus in this commit 1b2a937569096398257177107586295f33a1a467

Reply:
> Thinking about this some more, I'm not sure if we need to be moving focus at all. Using the "alert" role might be enough to ensure the error is read and accessible. Setting focus to the error may make sense if the previously focused element was removed during the error presentation, but if not, the focus moving may be more obtrusive than we expect.

That makes sense too.